v2/cf-nodejs-logging-support/src/lib/config/jwt-service.ts

```diff
@@ -0,0 +1,50 @@
+import Logger from "../logger/logger";
+
+const jwt = require("jsonwebtoken");
+const ENV_DYN_LOG_HEADER = "DYN_LOG_HEADER";
+const ENV_DYN_LOG_KEY = "DYN_LOG_LEVEL_KEY";
+const DEFAULT_DYN_LOG_LEVEL_HEADER = "SAP-LOG-LEVEL";
+
+export class JWTService {
```

Comment:
What is the motivation for using static methods and static attributes in this class?